templates/login.template.html

```diff
@@ -29,8 +29,5 @@ <h2 class="welcomeBox__title">Welcome to the upgraded network!</h2>
 
       <button type="submit" class="s-button loginBox__submit" ng-class="{'is-disabled': login.processing}" ng-disabled="login.processing">Sign in</button>
     </form>
-    <div class="legacyBox">
-      Looking for legacy login? <a href="https://launch.stellar.org/">Proceed here &raquo;</a>
-    </div>
```

Comment:
I think we shouldn't remove it. Some users haven't upgraded their accounts yet and there is no link to the old client. More: https://github.com/stellar/first-theme/issues/180